lib/favicons.js

```diff
@@ -54,11 +55,13 @@ function generateIcons (loader, imageFileStream, pathPrefix, query, callback) {
     appName: query.appName
   }, function (err, result) {
     if (err) return callback(err);
-    var html = result.html.filter(function (entry) {
-      return entry.indexOf('manifest') === -1;
```

Comment:
👍   I would suggest the following then.  Either remove `android` as a configurable option or fix the quality.  But, as it stands `android: true` has a bug since it doesn't inject HTML for the manifest.